tests/test_real_world.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import torch_ttnn
+import torchvision
+import ttnn
+import unittest
+import collections
+
+
+class TestRealWorld(unittest.TestCase):
+    def setUp(self):
+        # Open device 0
+        self.device: ttnn.Device = ttnn.open_device(device_id=0)
+
+    def tearDown(self):
+        # Close the device
+        ttnn.close_device(self.device)
+
+    @unittest.skip(
+        "Skip this test because it take 135 MB to download the ResNet18 model. Un-skip it if you want to test it."
+    )
+    def test_resnet(self):
```

Comment:
@kevinwuTT should this be removed, wdyt?

Reply:
This looks equivalent to https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/tests/models/resnet/test_resnet.py. If so then, it could be removed.